### ISSUE TITLE
Add ability to edit allowed sites through options UI

### ISSUE THREAD

**Issue by WesleyBranton** [mozilla/contain-facebook#550]
The users should be able to view and remove the allowed sites through the options UI. It's currently possible using the panel, but few users run with the Facebook Container browser action visible on their toolbar. Having that ability also built into the options UI would make it easier for users to find.

**Comment by maxxcrawford** [comment]
By Options UI, do you mean [this](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui)? Facebook Container does not have an options page set/created as of yet.

**Comment by WesleyBranton** [comment]
> By Options UI, do you mean [this](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui)? Facebook Container does not have an options page set/created as of yet.

Yes. Something more universal than the browserAction popup. Not every user has that saved to their toolbar.

**Comment by str** [comment]
I don't see any way to allow a site. What "browserAction popup"? I don't see any popup? What panel?
(I'm not being sarcastic, I really don't see a way to add a site) 

Please do add a way to add the site from a plugin config page.

**Comment by maxxcrawford** [comment]
@str The only way currently is through in-page UI on badged login buttons. (Example: [imgur.com](https:imgur.com/login)) 

![image](https://user-images.githubusercontent.com/2692333/71044862-809d6b80-20f8-11ea-92cf-c8f5b7df82d4.png)

Follow #422 for progress on this issue.

**Comment by maxxcrawford** [comment]
@WesleyBranton I've added a preferences page as an feature to discuss in our next roadmapping meeting. Thanks!

**Comment by str** [comment]
> @str The only way currently is through in-page UI on badged login buttons. 

[In pages that load images from instagram](https://www.tomorrowinayear.com/), there is no "facebook container icon"

![image](https://user-images.githubusercontent.com/123817/71046609-6b2b4000-20fe-11ea-87ec-6730b82f6a41.png)

Should I paste that in the #422 or here?

**Comment by maxxcrawford** [comment]
@str Actually…I would post this into #520

**Comment by RVAUser** [comment]
> 
> 
> @str The only way currently is through in-page UI on badged login buttons. (Example: [imgur.com](https:imgur.com/login))
> 
> ![image](https://user-images.githubusercontent.com/2692333/71044862-809d6b80-20f8-11ea-92cf-c8f5b7df82d4.png)
> 
> Follow #422 for progress on this issue.

This image pop-up does not appear when I click on the icon in my browser.  Instead, I get the following image 
![image](https://user-images.githubusercontent.com/59026639/71109442-32897600-2193-11ea-9291-7bafeb2ad3cc.png)

**Comment by maxxcrawford** [comment]
@RVAUser My pop-up was from clicking an on-page/in-content badge – not the panel/doorhanger.
